components/Tabs/src/MDCTabBar.h

```diff
@@ -152,3 +172,28 @@ IB_DESIGNABLE
 - (void)tabBar:(nonnull MDCTabBar *)tabBar didSelectItem:(nonnull UITabBarItem *)item;
 
 @end
+
+/**
+ Fixed configuration options for tab bars.
+
+ The tab bar copies the configuration passed at initialization, so all configuration changes must
+ be made before creating the tab bar. 
+*/
+@interface MDCTabBarConfiguration : NSObject <NSCopying>
```

Comment:
Can we mimic Apple and attach UIBarPositioningDelegate to MDCTabBarDelegate?
Doing so might eliminate the top/bottomConfiguration and allow us to move the remaining properties to MDCTabBar as IBInspectable @properties.